templates/base.html

```diff
@@ -51,7 +51,7 @@
               <a class="nav-link" href="/global">Global</a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="/Mine">Mine</a>
+              <a class="nav-link" href="/login">Mine</a>
```

Comment:
after the login, the user need to get in his page (with his house_id)

Reply:
Thats right, we talked about it in the meeting yesterday, the user needs to enter his house_id everytime he wants to get to his Mine Page